src/CoreBundle/private/library/classes/TCMSFields/TCMSFieldLookupMultiselectCheckboxesSelectFieldsFromTable.class.php

```diff
@@ -14,6 +14,27 @@
 /**/
 class TCMSFieldLookupMultiselectCheckboxesSelectFieldsFromTable extends TCMSFieldLookupMultiselectCheckboxes
 {
+    protected function GetMLTFilterQuery()
+    {
+        $filterQuery = parent::GetMLTFilterQuery();
+
+        // NOTE these conditions are _very_ specific to the current configuration of cms_field_conf
+
+        if (false !== \strpos($filterQuery, 'AS tabname') && false !== \strpos($filterQuery, 'cms_field_conf')) {
+            $orderClauseCheck = 'ORDER BY position ASC';
+            $orderPosition = \strpos($filterQuery, $orderClauseCheck);
+
+            // TODO "tabname" is the configuration "group field" or "group field column name" of the table - however that must be present in the custom list query.
+
+            if (false !== $orderPosition) {
+                // NOTE translations are handled later (EntityList?)
```

Comment:
Yes, the list is rendered with Tdb objects, so translations will be done there. But the query needs to handle the translations, too. Because the order by would not be alphabetical, if it is not  the base language. 

Something like:
ORDER BY ... `cms_field_conf`.'.$titleFieldWithTranslationSuffix.' ASC'

Reply:
Works in demo shop (with two languages).

Reply:
Strange. How? Maybe there is some magic fieldname replacement before running the query for all translated fields?

Reply:
Removed it now again.

And sort the data in code (only for checkboxes).